src/EditorFeatures/Core/Interactive/InteractiveSession.cs

```diff
@@ -86,6 +91,7 @@ internal sealed class InteractiveSession : IDisposable
             _workingDirectory = initialWorkingDirectory;
 
             _hostDirectory = Path.Combine(Path.GetDirectoryName(typeof(InteractiveSession).Assembly.Location)!, "InteractiveHost");
```

Comment:
>"InteractiveHost" [](http://example.com/codeflow?start=112&length=17)

Not your change, but this looks wrong. I think the subdir INteractiveHost shouldn't be added, but would need to debug to see what's going on

Reply:
This is what the directory looks like
```
C:\Users\dabarbet\AppData\Local\Microsoft\VisualStudio\17.0_99434ad3RoslynDev\Extensions\Microsoft\Roslyn Language Services\42.42.42.4242424\InteractiveHost
---> Core
        ---> CSharpInteractive.rsp
        ---> All the interactive host exe / dlls / resources.
---> Desktop
        ---> CSharpInteractive.rsp
        ---> All the interactive host exe / dlls / resources.
```